loops-and-arrays.js

```diff
@@ -41,6 +53,10 @@ const numbers = [2, 4, 5, 29, 38];
 // ==========================================
 
 const squares = [30, 2, 8, 24, 11];
+squaresLength=squares.length;
+for (let i = 0; i < squaresLength; i++) {
+    console.log(" Het volume van "+squares[i]+ " is "+ squares[i]*squares[i]*squares[i]);
```

Comment:
Dit is een detail, maar in de opdracht staat dat de waardes in de array overschreven moeten worden met "het volume van [x] is [y]. Jij hebt dit alleen gelogd in de console en niet de waardes aangepast.

Reply:
Dit heb je ook bij de andere deelopdrachten gedaan

Reply:
Ik heb al gewerkt met jouw opmerkingen